docs/src/man/split_apply_combine.md

```diff
@@ -13,6 +13,14 @@ In order to perform operations by groups you first need to create a `GroupedData
 object from your data frame using the `groupby` function that takes two arguments:
 (1) a data frame to be grouped, and (2) a set of columns to group by.
 
+!!! note
+
+    All operations described for `GroupedDataFrame` in this section of the manual
+    are also supported for `AbstractDataFrame` in which case it is considered as
+    being grouped on no columns (meaning it has a single group, or zero groups if it is empty).
+    The only difference is that in this case the `keepkeys` and `ungroup` keyword
+    arguments are not supported and a data frame is always returned.
```

Comment:
Not to be addressed by this PR, but it's weird that `select` and `transform` are documented in detail only in this section, which is about grouped operations.

Reply:
Agreed, but the point is that `select`, `transform`, and `combine` are now transparently handling data frames and grouped data frames in the same way - so maybe we just need to change the name of this section? As this note is about the case when "split" step is skipped.

Reply:
I have just renamed this section of the manual.